gpt_2_simple/gpt_2.py

```diff
@@ -573,32 +570,32 @@ def copy_checkpoint_to_gdrive(run_name='run1', copy_folder=False):
     """Copies the checkpoint folder to a mounted Google Drive."""
     is_mounted()
 
-    checkpoint_folder = os.path.join('checkpoint', run_name)
+    checkpoint_folder = f'checkpoint/{run_name}'
 
     if copy_folder:
-        shutil.copytree(checkpoint_folder, "/content/drive/My Drive/" + checkpoint_folder)
+        shutil.copytree(checkpoint_folder, f"/content/drive/My Drive/{checkpoint_folder}")
```

Comment:
I think you get the point by now. Fstrings are just overall *better* in these scenarios.